avaframe/com1DFA/deriveParameterSet.py

```diff
@@ -788,6 +788,8 @@ def checkRasterMeshSize(cfgSim, rasterFile, typeIndicator='DEM', onlySearch=Fals
     # if cell size of raster is different from desired meshCellSize - look for remeshed raster or remesh
     if np.abs(meshCellSize - headerRaster['cellsize']) > meshCellSizeThreshold:
         pathToRaster = geoTrans.remeshRaster(rasterFile, cfgSim, onlySearch=onlySearch)
+        if typeIndicator == 'FRIC':
```

Comment:
Why is this here, but not in the else?

And why is the pathToRaster relative in the case of FRIC?

Reply:
changed it to not be the relative one